exercism/python/pangram/pangram.py

```diff
@@ -1,8 +1,73 @@
 import re
+from string import ascii_lowercase as lowercase_letters
 
-
+# set
 def is_pangram(sentence):
-
     return (
         len(set(letter.lower() for letter in re.findall(r"[A-Za-z]", sentence))) == 26
```

Comment:
`26` is a magic number.

I think a better approach is to do:

```
set(letter.lower() for letter in re.findall(r"[A-Za-z]", sentence))) - set(lowercase_letters) == 0
```

or 

```
set(letter.lower() for letter in re.findall(r"[A-Za-z]", sentence))) == set(lowercase_letters)
```

or 

https://docs.python.org/3/library/stdtypes.html#frozenset.issubset

Reply:
I need to use A-Z in the regex otherwise I get a failing test. The input sentence can have upper or lowercase letters and I need to match on all of them, and then lowercase them. That's b/c a letter may only appear as a capital letter in the sentence, so I need to match on it in the regex in order to capture it in my set().

Make sense?

Reply:
Good comments -- thank you!

I made comments/questions back on your comments. Please look at them. 

Also, I'd like to get your thoughts to select 1 or 2 of the best solutions for a code reading example. 
My choices are labeled as: 
1. #set - 3
2. # array v2

People probably won't think of using ord(), which is what I use in `array v2`. But the set one is very clear and easy to understand. Thoughts?

Reply:
@amymhaddad On your first point above, I think it's a good idea to avoid using Python-specific things, so good call on avoiding `is subset`.

Reply:
For the code reading example, I like both the set and array approaches. I think the bit array approach is rarely used by programmers, so that may be a good one.